my-wireframe/css/index.css

```
@@ -0,0 +1,81 @@
+*{
+    box-sizing: border-box;
+}
+
+
+body{
+font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  font-size: 18px;
+  margin: 0;
+  padding: 0;
+}
+
+
+a{
+    padding: 10px;
+    background-color: black;
+    color: white;
+    border-radius: 8px;
+    text-decoration: none;
+}
+
+a:hover{
```

```suggestion
a:hover, a:focus{
```

Not all users are using a mouse 🪤

---

Thank you! I have made the changes.